monitor/sys/system.py

```diff
@@ -110,6 +110,8 @@ def service_boot():
         else:
             _logger.info("%s", result)
         events.system_status.trigger(msg="Loading servicing assets. Please wait.")
+        events.switch_mode.trigger(False)
```

Comment:
You named the event `mode_switch` but referenced it via `switch_mode` This line should be:
```python
events.mode_switch.trigger(False)
```